src/message/connection-endpoint.js

```diff
@@ -376,6 +380,12 @@ ConnectionEndpoint.prototype._registerAuthenticatedSocket  = function( socketWra
 		socketWrapper.sendMessage( C.TOPIC.AUTH, C.ACTIONS.ACK, [ messageBuilder.typed( userData.clientData ) ] );
 	}
 
+	if( userData.username != 'OPEN' ) {
+		var msg = messageBuilder.getMsg( C.TOPIC.PRESENCE, C.ACTIONS.PRESENCE_ADD, [ messageBuilder.typed( userData.username ) ] );
+		debugger;
+		setTimeout( this.onMessage( socketWrapper, msg ), 0);
```

Comment:
in node you can use

`process.nextTick( function );`

which is more efficient